examples/systemcalls/systemcalls.c

```diff
@@ -17,6 +33,16 @@ bool do_system(const char *cmd)
  *   or false() if it returned a failure
 */
 
+     int retvalue; //return value variable
+
+    retvalue = system(cmd); // fork + exec + wait
+    
+    if (retvalue == -1) // system error
```

Comment:
@Amey2904dash think about what would happen if the cmd returned with an error code other than -1 here.